ApartmentApp/app/views/apartments/_form.html.erb

```diff
@@ -27,12 +32,12 @@
   </div>
 
   <div class="field">
-    <%= f.label :street1 %>
+    <label for="apartment_street1">Address</label>
```

Comment:
Kept the same variable names, but changed the 'view' names of street1 and street2 to Address and Unit#